experiments/download_experts.py

```diff
@@ -0,0 +1,14 @@
+import urllib
```

Comment:
Why do we have this and `src/imitation/scripts/download_experts.py`? Surely only one is needed?

Reply:
My bad, I thought I had deleted this one

Reply:
The reason I moved to `imitation.scripts` was because I believed it would be important for integration script or test later. Not sure why now. This seems more appropriate for `experiments/` so I'll move it back.

Reply:
Ah, I think the reason I put it there was so that it would get linted and type checked for free. I'll quickly try adding lint and type check to `experiments/`.